lib/terraforming/resource/iam_policy_attachment.rb

```diff
@@ -65,7 +65,7 @@ def entities_for_policy(policy)
       end
 
       def iam_policies
-        @client.list_policies(scope: "Local").map(&:policies).flatten
+        @client.list_policies(scope: "All", only_attached: true).map(&:policies).flatten
```

Comment:
__[rubocop]__
* Line is too long. [88/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->